darc/darc.go

```diff
@@ -822,6 +839,7 @@ func (idkc IdentityX509EC) Verify(msg, s []byte) error {
 	if err != nil {
 		return err
 	}
+	log.Print("verifying 509 signature")
```

Comment:
remove? or change to a `log.Lvl*`?

Reply:
Also removed all other `log.Print` in the code.